my-react-app/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import CardList from './components/CardList.js'
+import './App.css';
+import axios from 'axios'
+
+
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      data: []
```

Comment:
This should be an object, the data we are getting back from the api is an object for a single user